corehq/apps/app_execution/forms.py

```diff
@@ -0,0 +1,50 @@
+from couchdbkit import NoResultFound, ResourceNotFound
+from crispy_forms import bootstrap as twbscrispy
+from crispy_forms import layout as crispy
+from django import forms
+
+from corehq.apps.app_execution.models import AppWorkflowConfig
+from corehq.apps.app_manager.dbaccessors import get_brief_app
+from corehq.apps.hqwebapp import crispy as hqcrispy
+from corehq.apps.users.models import CommCareUser
+
+
+class AppWorkflowConfigForm(forms.ModelForm):
+
+    class Meta:
+        model = AppWorkflowConfig
+        fields = "__all__"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.helper = hqcrispy.HQFormHelper()
+
+        self.helper.layout = crispy.Layout(
+            *self.fields.keys(),
+            hqcrispy.FormActions(
+                twbscrispy.StrictButton("Save",
+                                        type='submit',
+                                        css_class='btn-primary'),
+            ),
+        )
+
+    def clean(self):
+        self.final_clean_app_id()
+        self.final_clean_user_id()
+        print(self.cleaned_data)
+        return self.cleaned_data
+
+    def final_clean_user_id(self):
+        domain = self.cleaned_data.get("domain")
+        try:
+            CommCareUser.get_by_user_id(domain, self.cleaned_data.get("user_id"))
```

Comment:
Does it have to be a `CommCareUser`?  USH is working to have web users submitting data too.

Reply:
It doesn't have to be but for this use case I thought it made more sense for it to be a mobile user since they are easier to manage.